practice-two/src/style/section/enterprise.css

```diff
@@ -0,0 +1,68 @@
+/* START ENTERPRISE */
+.section-enterprise {
+  text-align: center;
+  background: linear-gradient(180deg, #F7F7F7 0%, rgba(247, 247, 247, 0) 100%);
+}
+
+.general-content {
+  max-width: 1240px;
+  padding: 0 20px;
+}
+
+.section-enterprise .content {
+  font-weight: (--font-weight-xs: 400;);
+}
+
+.secondary-btn {
+  background: var(--primary-color-100);
+  color: var(--secondary-color-100);
+  border-radius: 8px;
+}
+
+.secondary-btn:hover {
+  background-color: var(--secondary-color-100);
+  color: var(--primary-color-100);
+} 
+
+.mr-btn {
+  margin: 38px auto 51px;;
+}
+
+@media (max-width: 915px) {
```

Comment:
please add comment for each break point. is this style applied for tablet or mobile?